packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/CheckIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { borders, CSSColor } from '@trezor/theme';
+import { Icon } from '@trezor/components';
+
+const IconWrapper = styled.div<{ $background: CSSColor }>`
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    background: ${({ $background }) => $background};
+    border-radius: ${borders.radii.full};
+`;
+
+type CheckIconProps = {
+    isTorContainerHoveredOrFocused?: boolean;
+};
+
+export const CheckIcon = ({ isTorContainerHoveredOrFocused = false }: CheckIconProps) => {
```

Comment:
According to component name it looks very generic but according to props it looks like it's very specific. I suggest to rename it somehow to make it more readable what should happen here.

Reply:
Yes, of course. I just left the original name here for now, while figuring out the better name for the component.